pulp_ansible/app/models.py

```diff
@@ -227,6 +229,60 @@ class CollectionRemote(Remote):
     TYPE = "collection"
 
     requirements_file = models.TextField(null=True, max_length=255)
+    auth_url = models.CharField(null=True, max_length=255)
```

Comment:
I was thinking this may need to be split info two Remote subclasses. One for the 'v2' and galaxy.ansible.com API, and another for Automation-hub / cloud.redhat.com / galaxy_ng.

Partly because "community" galaxy.ansible.com and cloud.redhat.com need different auth tokens.

cloud.redhat.com needs:
-  the 'auth_url' (pointing at https://sso.redhat.com/auth/realms/redhat-external/protocol/openid-connect/token SSO server)
- The value of "token" is a "refresh_token" in from SSO  (the token a user gets from 'GET API TOKEN' on the cloud.redhat.com automation-hub ui).
- The token received from SSO is used in 'Autorization: Bearer <token here>'

community / galaxy.ansible.com / maybe standalone galaxy_ng
- No 'auth_url' is required.   For ansible-galaxy client the presence of a auth_url is what determines if SSO style auth is used and what format the authorization token takes.
- The value of "token" is a drf auth token
- The token is used in an 'Authorization: Token <token here>'  header.

Reply:
name wise, I was thinking more or less "CollectionRemote" for v2 / community galaxy.ansible.com remotes since it exists already.

And 'AutomationHubRemote' for Remotes for talking to automation-hub / cloud.redhat.com.

Reply:
The differences in the technologies of the tokens (to me) motivates splitting up the Remote into two.